ensime_shared/ensime.py

```diff
@@ -48,10 +48,12 @@ def __init__(self, vim):
         # race condition of autocommand handlers being invoked as they're being
         # defined.
         self._vim = vim
+        self._editor = Editor(self._vim)
         self.clients = {}
 
+    @property
     def using_server_v2(self):
-        """Whether user has configured the plugin to use ENSIME v2 protocol."""
+        """bool: Whether user has configured the plugin to use ENSIME v2 protocol."""
```

Comment:
Nvm, it's an undocumented setting so only people knowing what they do should be using it.